dashboard/modules/job/job_manager.py

```diff
@@ -531,10 +533,11 @@ async def submit_job(
                 )
             supervisor = self._supervisor_actor_cls.options(
                 lifetime="detached",
-                name=self.JOB_ACTOR_NAME_TEMPLATE.format(job_id=submission_id),
+                name=JOB_ACTOR_NAME_TEMPLATE.format(job_id=submission_id),
                 num_cpus=0,
                 scheduling_strategy=scheduling_strategy,
                 runtime_env=self._get_supervisor_runtime_env(runtime_env),
+                namespace=SUPERVISOR_ACTOR_RAY_NAMESPACE,
```

Comment:
add the namespace here, because the GCS::GetNamedActor  seems to be unable to get `ActorInfo`  without namespace